tests/console/java.pm

```diff
@@ -39,7 +39,7 @@ sub run {
     # java-10-openjdk & java-1_8_0-ibm  -> Legacy
     my $cmd = 'install --auto-agree-with-licenses ';
     $cmd .= (is_sle('15+') || is_leap) ? 'java-11-openjdk* java-1_*' : 'java-*';
-    zypper_call($cmd, timeout => 1500);
+    zypper_call($cmd, timeout => 2000);
```

Comment:
I wonder if at this point, the max job timeout shouldn't be changed

Reply:
This could be handled later, if we encounter some problems.

Reply:
indeed